web/server/vue-cli/src/components/Report/SelectReviewStatus.vue

```diff
@@ -57,6 +57,33 @@
 
       <v-card-text class="pa-0">
         <v-container>
+          <v-alert
+            v-if="reportsReviewedInSource.length != 0"
+            dense
+            type="error"
+          >
+            <p>
+              Review status of the following reports are given as source code
+              comment. Their review status will not change.
+            </p>
+
+            <div
+              v-for="r in reportsReviewedInSource"
+              :key="parseInt(r.reportId)"
+              @click="selectSameReport(r.reportId)"
+            >
+              <detection-status-icon
```

Comment:
It would be good to show review statuses for these reports too.

Also I think it would be useful to show it in the `Found in` drop-down window:
![image](https://user-images.githubusercontent.com/6695818/152505811-1dea030a-5050-45e6-9f5d-40e709ce7d60.png)